src/Soql/AbstractSoqlDriver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codelicia\Soql;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\DriverException;
+use Doctrine\DBAL\Driver\ExceptionConverterDriver;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Platforms\MySqlPlatform;
+
+abstract class AbstractSoqlDriver implements Driver, ExceptionConverterDriver
```

Comment:
Is this abstraction needed? Will this have to deal with multiple platform versions and all?

Reply:
Well, we have different API versions all the way up to `v46`. But I'm indeed not sure if that is the way to work with it. May I have the opinions of @EHER and @airtonzanon here as well?